### ISSUE TITLE
python module doesn't prefer "py -3" over %LOCALAPPDATA%\Microsoft\WindowsApps\python3.exe

### ISSUE THREAD

**Issue by barcharcraz** [mesonbuild/meson#5886]
if you have python3 installed (but not on the path, or only on the path as "python") and you are on a recent version of windows 10 meson's python module fails because it finds `%LOCALAPPDATA%\Microsoft\WindowsApps\python3.exe` which is a stub that launches the windows store, not actual python3. It then fails and does not continue.

**Comment by dcbaker** [comment]
that is.... special

**Comment by barcharcraz** [comment]
yeah I am not super happy. tar.exe being bsdtar is at least reasonable. This feels like the saga of the curl -> Invoke-WebRequest alias in powershell.

**Comment by bonzini** [comment]
Fixed by f6a842821b64653de042633e714af3c5a5326c29
